src/System.Data.SqlClient/tests/Tools/TDS/TDS/FeatureExtAck/TDSFeatureExtAckFederatedAuthenticationOption.cs

```diff
@@ -137,4 +137,4 @@ public override bool Inflate(Stream source)
             return true;
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Just realized my editor automatically added new lines to the end of 2 of these files, I will amend. 😅

Reply:
>Just realized my editor automatically added new lines to the end of 2 of these files, I will amend.

That doesn't need to be reverted.  It is a ok thing to fix.

Reply:
Alright, wasn't sure! Maybe this should be included in the coding style guide, or maybe it already falls under the general rule "use Visual Studio defaults".

Reply:
There should be an extra end-of-line in the file or else Git complains.
![image](https://user-images.githubusercontent.com/7684835/43041629-061df7fc-8d1b-11e8-98bc-62c72f11e4d1.png)

The fact that your editor fixed it for you is a bonus.

Generally, we would prefer not to see a lot of PRs where people are just fixing a large number of files with end-of-line problems.  But fixing it in a few files where you are fixing other problems is fine for the PR.

The coding style guide tries to include most things but it is difficult to add everything into a written guide.